tests/module_2/test_statistical_analysis.py

```diff
@@ -0,0 +1,57 @@
+
+import sys
+sys.path.append('/home/oscar/zrive-ds')
+
+import unittest
+import numpy as np
+import pandas as pd
+from scipy.stats import anderson
+from src.module_2.statistical_analysis import *
+
+class TestNormalityShapiroFunction(unittest.TestCase):
```

Comment:
This tests aren't typically needed since you are basically testing an external package. They could have been skipped.